src/less/dropdown/base/dropdown.less

```diff
@@ -111,6 +113,20 @@ button.fake-menu__item {
     &:not(:last-child) {
         margin-bottom: 1px;
     }
+
+    &:hover,
```

Comment:
Question, should we add a :focus selector for either this one or active? For accessibility

Reply:
We might, but we will want to address that outside of the refactor, likely as part of this issue: #633.